### DISCUSSION TITLE
How to expand a ScrollArea without a fixed size

### DISCUSSION THREAD

**Discussion by floppyhammer** [emilk/egui#3016] [Q&A]
So I have a ScrollArea like this.

<img width="212" alt="image" src="https://github.com/emilk/egui/assets/28705694/5bb88aca-4c2d-4170-8ddc-135e099d134f">

Is there any way I can expand it as follows, and it can fit dynamically when I resize the window?

<img width="212" alt="Screenshot 2023-05-23 102526" src="https://github.com/emilk/egui/assets/28705694/470fffd8-dd59-4852-b6fa-2088d0fca97e">

**Comment by JumpyLionnn** [comment]
You can try to set auto shrink to false.
see, https://docs.rs/egui/latest/egui/containers/scroll_area/struct.ScrollArea.html#method.auto_shrink
